### PR TITLE
Badge Magic PCB layout

### LINKED ISSUE #13: Design a new board
For now, we have just a schematic for firmware reference. We will need a new PCB with more features added, easier debugging, and better design.

### PR DESCRIPTION
**Pull request by kienvo**
Resolves #13

### CODE REVIEW

**Comment by mariobehling** [comment]
@kienvo @fcartegnie @CloudyPadmal What battery could we add here?

File: docs/bill-of-materials.csv [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+"J2","USB_C_Receptacle_USB2.0_16P","https://www.lcsc.com/datasheet/lcsc_datasheet_2312191005_G-Switch-GT-USB-7047A_C963216.pdf","BadgeMagic:USB_C_Receptacle_GCT_USB4110","1","0.1577"," GT-USB-7047A","USB 2.0-only, 16P, SMD, Type-C Receptacle connector. There are 2 mechanical pin for positioning, should be chop off before soldering."
+"KEY1,KEY2","button","https://www.lcsc.com/datasheet/lcsc_datasheet_1811011023_HYP--Hongyuan-Precision-1TS003A-1800-3500A-CT_C255806.pdf","BadgeMagic:1TS003A-1800-3500A-CT","2","0.0431"," 1TS003A-1800-3500A-CT","SPST SMD,2x4mm Tactile Switches"
+"MK1"," WM-64C","https://www.lcsc.com/datasheet/lcsc_datasheet_2312271240_PANASONIC-WM-64C_C19458520.pdf; https://datasheet.octopart.com/WM-64PCX-Panasonic-datasheet-13695803.pdf","BadgeMagic:WM-64PC","1","2.0725"," WM-64C","Electret microphone"
```
Comment (mariobehling) [inline docs/bill-of-materials.csv:7]: Please remove spacing in front of WM-64C

**Comment by mariobehling** [comment]
Can you find a cheaper microphone possibly?

File: docs/bill-of-materials.csv (hunk 1 above)
Comment (kienvo) [reply docs/bill-of-materials.csv]: Resolved

**Comment by kienvo** [comment]
> Can you find a cheaper microphone possibly?

`GMI4015-2C-30db` seems good with a smaller package.

**Comment by mariobehling** [comment]
Please also update the schematics PDF and add a front and back image to the Readme similar to the PSLab project.

File: docs/images/BadgeMagic-front.png
Comment (mariobehling) [inline docs/images/BadgeMagic-front.png:1]: I think we do not need the silkscreen text behind the battery. Therefore please check if you could change to something like the following.

Please also
* use the all caps OPNTEC logo.
* add the version number and version date
* reduce the width of the white battery (or in this picture it is the height). The batteries are usually slightly smaller.

Please check if the QR code is still big enough to be recognized by a cam scanner.

BadgeMagic

**Comment by kienvo** [comment]
Front side:



Back side:

**Review by CloudyPadmal — APPROVED** [review]

File: docs/images/BadgeMagic-front.png
Comment (Orbiter) [reply docs/images/BadgeMagic-front.png:1]: The branding on the PCB is great!

**Pull request merged**